radio/src/opentx.h

```diff
@@ -27,6 +27,66 @@
 #include "opentx_helpers.h"
 #include "touch.h"
 
+//OW
+#define OWVERSIONSTR  "v27"
```

Comment:
Please a more “neutral” name.

Reply:
when you give your ok I'll sure will clean up

Reply:
done

Reply:
Thx!